ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -246,10 +246,17 @@ def getGroupsWithVOMSAttribute( vomsAttr ):
 def getVOs():
   """ Get all the configured VOs
   """
-  voName = getVO()
-  if voName:
-    return S_OK([ voName] )
-  return gConfig.getSections( '%s/VO' % gBaseRegistrySection )
+  #voName = getVO()
+  #if voName:
+  #  return S_OK([ voName] )
+  #return gConfig.getSections( '%s/VO' % gBaseRegistrySection )
+  vos = list()
+  if not gConfig.getSections( '/Registry/VO' )['OK']:
+    return False
+  for dirVO in gConfig.getSections( '/Registry/VO' )['Value']:
+    if gConfig.getOption( '/Registry/VO/%s/VOMSName' % dirVO )['OK']:
+      vos.append(gConfig.getOption( '/Registry/VO/%s/VOMSName' % dirVO )['Value'])
+  return vos
```

Comment:
This is not OK. The previous implementation was returning an S_OK/S_ERROR dictionary, and this new one returns either `False` or `list`.